bears/haskell/HaskellLintBear.py

```diff
@@ -38,11 +38,15 @@ def process_output(self, output, filename, file):
         output = json.loads(output)
 
         for issue in output:
-            assert issue['startLine'] == issue['endLine']
             diff = Diff(file)
+            from_lines = issue['from'].splitlines()
+            to_lines = issue['to'].splitlines()
+            assert len(from_lines) == len(to_lines)
+            for other_lines in range(1, len(from_lines)):
+                assert from_lines[other_lines] == to_lines[other_lines]
```

Comment:
you don't use "`other_lines`" laer, do you? You have `from_lines[0]` only, so I wouldn't `assert` here, is not necessary imo :)